modules/openvpn_status/openvpn_status_test.go

```diff
@@ -0,0 +1,108 @@
+package openvpn_status
+
+import (
+	"fmt"
+	"io/ioutil"
+	"os"
+	"testing"
+
+	"github.com/netdata/go.d.plugin/agent/module"
+	"github.com/netdata/go.d.plugin/pkg/matcher"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+)
+
+const (
+	v1statusLog = `OpenVPN CLIENT LIST
+Updated,2022-04-17 00:33:28
+Common Name,Real Address,Bytes Received,Bytes Sent,Connected Since
+gofle,192.168.122.135:58999,19265,261631,2022-04-17 00:33:19
+client_bsd2,192.168.122.136:11445,2752,3545,2022-04-17 00:33:04
+ROUTING TABLE
+Virtual Address,Common Name,Real Address,Last Ref
+10.8.0.2,gofle,192.168.122.135:58999,2022-04-17 00:33:27
+10.8.0.3,client_bsd2,192.168.122.136:11445,2022-04-17 00:33:04
+GLOBAL STATS
+Max bcast/mcast queue length,2
+END
+`
+	v2statusLog = `TITLE,OpenVPN 2.5.5 x86_64-pc-linux-gnu [SSL (OpenSSL)] [LZO] [LZ4] [EPOLL] [PKCS11] [MH/PKTINFO] [AEAD] built on Mar 22 2022
+TIME,2022-03-31 11:47:46,1648716466
+HEADER,CLIENT_LIST,Common Name,Real Address,Virtual Address,Virtual IPv6 Address,Bytes Received,Bytes Sent,Connected Since,Connected Since (time_t),Username,Client ID,Peer ID,Data Channel Cipher
+CLIENT_LIST,gofle,192.168.122.135:59523,10.8.0.2,,19265,261631,2022-03-31 11:47:34,1648716454,UNDEF,10,0,AES-128-GCM
+CLIENT_LIST,client_bsd2,192.168.122.136:20053,10.8.0.3,,2752,3545,2022-04-17 00:02:39,1650142959,UNDEF,0,0,AES-128-GCM
+HEADER,ROUTING_TABLE,Virtual Address,Common Name,Real Address,Last Ref,Last Ref (time_t)
+ROUTING_TABLE,10.8.0.2,gofle,192.168.122.135:59523,2022-03-31 11:47:37,1648716457
+ROUTING_TABLE,10.8.0.3,client_bsd2,192.168.122.136:20053,2022-04-17 00:02:39,1650142959
+GLOBAL_STATS,Max bcast/mcast queue length,1
+END`
+	v3statusLog = `TITLE	OpenVPN 2.5.5 x86_64-pc-linux-gnu [SSL (OpenSSL)] [LZO] [LZ4] [EPOLL] [PKCS11] [MH/PKTINFO] [AEAD] built on Mar 22 2022
+TIME	2022-03-31 11:53:29	1648716809
+HEADER	CLIENT_LIST	Common Name	Real Address	Virtual Address	Virtual IPv6 Address	Bytes Received	Bytes Sent	Connected Since	Connected Since (time_t)	UsernameClient ID	Peer ID	Data Channel Cipher
+CLIENT_LIST	gofle	192.168.122.135:49199	10.8.0.2		19265	261631	2022-03-31 11:53:19	1648716799	UNDEF	0	0	AES-128-GCM
+CLIENT_LIST     client_bsd2     192.168.122.136:21748   10.8.0.3       2001:0db8:85a3:0000:0000:8a2e:0370:7334         2752   3545  2022-04-16 20:48:27     1650131307      UNDEF   624     0       AES-128-GCM
+HEADER	ROUTING_TABLE	Virtual Address	Common Name	Real Address	Last Ref	Last Ref (time_t)
+ROUTING_TABLE	10.8.0.2	gofle	192.168.122.135:49199	2022-03-31 11:53:24	1648716804
+ROUTING_TABLE   10.8.0.3        client_bsd2     192.168.122.136:21748   2022-04-16 20:48:27     1650131307
+GLOBAL_STATS	Max bcast/mcast queue length	0
+END`
+)
```

Comment:
Let's have them as txt files under `testdata/`. That is a common pattern we use, also it will make testing easier for you since it will be closer to the real case (reading files).